board/piksiv3/rootfs-overlay/etc/init.d/S86skylark0_setup

```diff
@@ -0,0 +1,26 @@
+#!/bin/sh
+
+name="skylark_setup"
+
+start() {
+  # Setup DNS
+  # TODO (mookerji): Move to a template command?
```

Comment:
Agree about the conf and the timestamp thing. My understanding is that we don't have time. Perhaps we should add ... NTP ?